### ISSUE TITLE
[Copier] - Add Support for Embeds? Copy as json

### ISSUE THREAD

**Issue by XxUnkn0wnxX** [Strencher/BetterDiscordStuff#121]
1st I know there is this plugin here: https://github.com/GR0SST/CopyRawEmbed/blob/main/CopyRawEmbeds.plugin.js
it was a part of the plugin repo, however, it's gone now.

if possible could this or something like this be implemented into the **Copier** Plugin?
it's the only thing missing.

Example Source Embed (this is how it should output more or less..) [JSON]
```json
{
	"footer": {
		"text": "XxUnkn0wnxX",
		"icon_url": "https://cdn.discordapp.com/attachments/873210537423560714/904402483781206026/Main-Current.jpg"
	},
	"author": {
		"name": "Slayers Ice Realm",
		"icon_url": "https://cdn.discordapp.com/attachments/873210537423560714/886798919869272064/slayer.gif"
	},
	"color": 11535107,
	"description": "**React To <a:slayer_xmassnowflake:923817502373453856> to obtain the <@&923812145542623262> Role!**\nOnly available till __New Years 2021!__",
	"title": "<a:slayer_xmastree:923817502922903572> Xmas Role <a:slayer_xmastree:923817502922903572>"
}
```

This is very useful for currently made embeds which staff forgot to save source for, or copy other servers embeds & use them as a template for our own.

<img width="715" alt="Copier" src="https://user-images.githubusercontent.com/2242880/147804097-e4a1948a-6688-437a-b1b1-48cef9b661be.png">

Edit: I hope this gets added since Discord broke the original plugin & i'm unsure of the original dev even cares about BD anymore...

**Comment by XxUnkn0wnxX** [comment]
@Strencher Just tested it hover, it doesn't output in a friendly format where I can just tweak/look at it via carl's embed editor or other embed editor tools out there.

[Video Demo](https://www.dropbox.com/s/z4gyy25wkqctcqi) of what I mean your raw embed format Vs the other one..
[Embed Examples.txt](https://github.com/Strencher/BetterDiscordStuff/files/8036809/Embed.Examples.txt) - the one I used from the video, same embeds but your raw output vs the one that worked with carl embed editor..

**Comment by HypedDomi** [comment]
Copier copys the Embed Object. Carl uses the webhook format. Strencher could add a setting in the plugin where you could set the format to work with webhook

**Comment by XxUnkn0wnxX** [comment]
@Strencher I noticed that this feature does not work in the current build of copier 1.6.0, did you forget to add it back in?

**Comment by Strencher** [comment]
Yeah that's possible. I think I kinda left it out because I was too lazy to write a embed restore function that gets it back to it's original structure.

**Comment by XxUnkn0wnxX** [comment]
I really liked the way this plugin worked, however, it's been dead for a long time: https://github.com/GR0SST/CopyRawEmbed/blob/main/CopyRawEmbeds.plugin.js

it was perfect i could copy embeds & paste them right into carlbot & dischook with some tweaks & it worked more or less.

**Comment by ArefinB** [comment]
> I really liked the way this plugin worked, however, it's been dead for a long time: https://github.com/GR0SST/CopyRawEmbed/blob/main/CopyRawEmbeds.plugin.js
> 
> it was perfect i could copy embeds & paste them right into carlbot & dischook with some tweaks & it worked more or less.
> 
> so if you can make it spit out in the format that calbot & dischook prints out their embeds via their backup options for carlbot the command was: `!embedsource <message id> [channel]` & it would print out a json file like this:
> 
> ```json
> {
>     "fields":[
> 
>     ],
>     "color":11731024,
>     "type":"rich",
>     "description":"__**Age Roles**__\n\n**<:slayer_18_1:890501718914256976>\ufe19<@&916620571255668806>**\n\n**<:slayer_18_2:890502228375396392>\ufe19<@&916620687232360488>**\n\nReact to the **Emoji** to get the `role` you want!"
> }
> ```
> 
> & this is dischook's json export:
> 
> ```json
> {
>   "content": null,
>   "embeds": [
>     {
>       "description": "__**Age Roles**__\n\n**<:slayer_18_1:890501718914256976>︙<@&916620571255668806>**\n\n**<:slayer_18_2:890502228375396392>︙<@&916620687232360488>**\n\nReact to the **Emoji** to get the `role` you want!",
>       "color": 11731024
>     }
>   ],
>   "attachments": []
> }
> ```

I used this plugin too. The only option I see is Copier now. But sadly it cannot copy RAW from embeds.
